contrib_bots/lib/translate.py

```diff
@@ -0,0 +1,95 @@
+# See zulip/contrib_bots/lib/readme.md for instructions on running this code.
+from __future__ import print_function
+import sys
+import logging
+try:
+    from mstranslator import Translator
+except ImportError:
+    logging.error("Dependency Missing!")
+    sys.exit(1)
+
+#Uses mstranslator module
```

Comment:
Please, leave a space after the `#` when writing inline comments ([PEP8 guideline](https://www.python.org/dev/peps/pep-0008/#inline-comments)).